netty/shaded/build.gradle

```diff
@@ -18,7 +18,6 @@ dependencies {
             project(':grpc-testing-proto'),
             project(':grpc-testing'),
             libraries.truth
-    shadow project(':grpc-core')
```

Comment:
I figured out why this is here: it is necessary for the pom to include a dependency on grpc-core.

Reply:
I see. Changed back.